app/src/editor/canvas/state/index.js

```diff
@@ -484,11 +484,6 @@ function disconnectInput(canvas, portId) {
 
         delete newPort.sourceId
 
-        // ethereum contract input
-        if (newPort.type === 'EthereumContract' && newPort.value) {
-            delete newPort.value
-        }
-
```

Comment:
This part is needed (or the value has to be cleared somehow when disconnected), otherwise the events are not cleared:
![ezgif-2-5b34400d2c09](https://user-images.githubusercontent.com/1064982/62139143-7c131a80-b2f1-11e9-9ec7-55a694a38eb1.gif)